static/src/js/components/DeleteModal/DeleteModal.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Button from 'react-bootstrap/Button'
+import Modal from 'react-bootstrap/Modal'
+
+/**
+ * @typedef {Object} DeleteModalProps
+ * @property {Boolean} show Should the modal be open.
+ * @property {Function} closeModal A function to close the modal.
+ * @property {Function} onDelete A callback function triggered when the user selects `Yes`.
+ */
+
+/**
+ * Renders a DeleteModal component
+ *
+ * @component
+ * @example <caption>Render a DeleteModal</caption>
+ * return (
+ *   <DeleteModal
+ *      show={showDeleteModal}
+ *      closeModal={handleClose}
+ *      onDelete={handleDelete}
+ *   />
+ * )
+ */
+
+// TODO check if the record being deleted is in the current provider,
+// if not show the messages to change it
+const DeleteModal = ({
```

Comment:
Call it Modal
pass a message
action as array, (forEach though them
{
  label: "yes"
  variant: 'primary'
  onClick: ()=>{some fucntion}
}